provider/virtual_network/network_base.py

```
@@ -472,22 +496,43 @@ def _get_access_info(netperf_address):
 
     c_func, c_ip, c_session = _get_access_info(netperf_client)
     s_func, s_ip, s_session = _get_access_info(netperf_server)
-
+    if not utils_package.package_install("netperf", s_session):
+        raise exceptions.TestError("Unable to install netperf in the server host!")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)